contracts/teller-network/ArbitratorLicense.sol

```diff
@@ -0,0 +1,93 @@
+
+pragma solidity ^0.5.8;
+
+import "./License.sol";
+
+/**
+* @title ArbitratorLicense
+* @dev Contract for management of an arbitrator license
+*/
+contract ArbitratorLicense is License{
+
+    enum RequestStatus {AWAIT,ACCEPTED,REJECTED}
+
+    struct Requests{
+        address seller;
+        address arbitrator;
+        RequestStatus status;
+    }
+
+	struct ArbitratorLicenseDetails {
+        uint id; 
+        bool acceptAny; // accept any seller
+        address[] accepted; // addresses of accepted sellers
+    	bool isActive; // is license active?
+    }
+
+    mapping(address => ArbitratorLicenseDetails) arbitratorlicenseDetails;
+
+    Requests[] public requests;
+    mapping(uint => uint[]) public requestsById;
+
+    event ArbitratorRequested(uint id, address seller, address arbitrator);
+    event ArbitratorLicensed(uint id, bool acceptAny);
+    event RequestAccepted(address arbitrator, address seller);
+
+    /**
+     * @notice Buy an arbitrator license
+     * @param _acceptAny indicates does arbitrator allow to accept any seller/choose sellers 
+     */
+    function buyLicense(bool _acceptAny) public {
+    	uint _id = buy();
+    	address[] memory addresses;
+
+        arbitratorlicenseDetails[msg.sender] = ArbitratorLicenseDetails({
+            id: _id,
+            acceptAny: _acceptAny,
+            accepted: addresses,    
+ 			isActive:  true			
```

Comment:
What's the difference between `isActive` and `isLicenseOwner`?

Reply:
the idea is that Arbitrator license acts as an additional layer on top of a regular license, so isLicense owner would return true/false if address has a license, and isActive would return true/false if address has an arbitrator license